entries/downstream-version-100/game.json

```diff
@@ -0,0 +1,21 @@
+{
+    "title": "Downstream (Version 1.0.0)",
+    "slug": "downstream-version-100",
+    "developer": "DeltaGardner",
+    "platform": "GB",
+    "typetag": "homebrew",
+    "screenshots": [
+        "placeholder.png"
```

Comment:
You don't need to put (and reference) the placeholder screenshot, it's up to the frontend to render it when no screenshots are available for a entry

Reply:
Should be fixed!